tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -190,7 +190,7 @@ module.exports = {
         .waitForOutstandingAjaxCalls()
         .waitForElementNotPresent('@addShareSaveButton')
     },
-    saveCollaboratorPermission: function () {
+    saveCollaboratorEditForm: function () {
```

Comment:
I think a better name would be `saveChanges`

Reply:
done! :+1: